docs/HowToUseSmartFSDumpVisualizer.md

```diff
@@ -14,26 +20,37 @@ To run SDV, please download and install Java 8 (or the latest Java) to a target
 
 >**Note** To use TizenRT IDE together, Java 8 should be installed.
 
-SDV can be released for Windows, Linux (Ubuntu), and Mac OS.
+SDV can be released for Linux (Ubuntu), Windows (Some functions are constrained), and Mac OS (Possible but not yet).
 
 After downloading the SDV release (zip/tar.gz) from the following links,
+[Ubuntu](../tools/fs/FS_Dump_Parser/Release/SDV_v2.0_20200210_Ubuntu.tar.gz)
 [Windows](../tools/fs/FS_Dump_Parser/Release/SDV_v1.1_20190903_Windows.zip)
-[Ubuntu](../tools/fs/FS_Dump_Parser/Release/SDV_v1.1_20190903_Ubuntu.tar.gz)
 
 Decompress the release file and execute SDV.exe/SDV.
```

Comment:
What's the meaning of `SDV.exe/SDV`?

Reply:
Those are launch files for Windows/Ubuntu.